vignettes/predict.Rmd

```diff
@@ -152,3 +152,24 @@ of $\theta$ and variance-covariance matrix of $\theta$ are provided.
 Using the generated $\theta_{new}$, we then obtain the $\beta_{new}$ and its variance-covariance matrix, with $\theta_{new}$ and
 the data used in fit. Then we simulate once with `simulate(method = "conditional", beta = beta_new, theta = theta_new)`.
 Pool all the repetition together and we get the marginal simulation results.
+
+## Comparison with `SAS`
+
+In `SAS`, from `proc mixed`, we are able to generate predictions using `outp` argument in `model` statement.
+An example will be
+
+```{sas eval=FALSE}
+PROC MIXED DATA = fev_data method=reml;
+  CLASS RACE(ref = 'Asian') AVISIT(ref = 'VIS4') SEX(ref = 'Male') ARMCD(ref = 'PBO') USUBJID;
+  MODEL FEV1 = ARMCD / ddfm=Satterthewaite solution chisq outpm=predm outp=pred;
```

Comment:
```suggestion
  MODEL FEV1 = ARMCD / ddfm=Satterthewaite solution chisq outp=pred;
```
because you only mention `outp` above and I understand we did not compare with `outpm` so far?